uw-frame-static/superstatic.json

```diff
@@ -0,0 +1,10 @@
+{
+  "rewrites" : [
+    {"source":"/settings", "destination" : "index.html"},
+    {"source":"/notifications", "destination" : "index.html"},
+    {"source":"/features", "destination" : "index.html"},
+    {"source":"/about", "destination" : "index.html"},
+    {"source":"/server-error", "destination" : "index.html"},
+    {"source":"/access-denied", "destination" : "index.html"}
+  ]
+}
```

Comment:
True. In the end each app is going to define there own routes, but it is handy to have the defaults.